helpers/helpers.go

```diff
@@ -1,11 +1,47 @@
 package helpers
 
 import (
+	"database/sql"
 	"encoding/json"
 	"net/http"
+
+	"github.com/david1312/tsaving/models"
 )
 
 func HTTPError(w http.ResponseWriter, status int, errorMessage string) {
 	w.WriteHeader(status)
 	json.NewEncoder(w).Encode(map[string]string{"error": errorMessage})
 }
+
+func CheckBalance(target string, accNumber string, amount int, db *sql.DB) (status bool) {
+	if target == "MAIN" {
+		saldoasal, err := models.GetBalanceAcc(accNumber, db) // println(err.Error())
+		//cek dapet engga balancenya
+		if err != nil {
+			status = false
```

Comment:
No need to set the status to false since boolean default value in golang is false.